python/tvm/topi/nn/depthwise_conv2d.py

```diff
@@ -284,9 +295,13 @@ def depthwise_conv2d_nhwc(Input, Filter, stride, padding, dilation, out_dtype=No
                     j * stride_w + dj * dilation_w,
                     idxdiv(c, channel_multiplier),
                 ].astype(out_dtype)
-                * Filter[
-                    di, dj, idxdiv(c, channel_multiplier), idxmod(c, channel_multiplier)
-                ].astype(out_dtype)
+                * Filter.__getitem__(
```

Comment:
The explicit `__getitem__` isn't required.  `Filter[x,y,z]` is syntactic sugar for `Filter[(x,y,z)]`.  If you already have a tuple, then `Filter[tuple( ... )]` can be called.

Reply:
Oh I see, I have changed this as your recommendation.